nms/app/packages/magmalte/app/views/metrics/Explorer.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import ActionTable from '../../components/ActionTable';
+import CardTitleRow from '../../components/layout/CardTitleRow';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import ExploreIcon from '@material-ui/icons/Explore';
+import Grafana from '../../components/Grafana';
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+
+import {LteMetricsTable} from './LteMetricsData';
+import {colors, typography} from '../../theme/default';
+import {makeStyles} from '@material-ui/styles';
+
+const TITLE = 'Metrics Explorer';
+
+const useStyles = makeStyles(theme => ({
+  dashboardRoot: {
+    margin: theme.spacing(5),
+  },
+  appBarBtn: {
+    color: colors.primary.white,
+    background: colors.primary.comet,
+    fontFamily: typography.button.fontFamily,
+    fontWeight: typography.button.fontWeight,
+    fontSize: typography.button.fontSize,
+    lineHeight: typography.button.lineHeight,
+    letterSpacing: typography.button.letterSpacing,
+
+    '&:hover': {
+      background: colors.primary.mirage,
+    },
+  },
+  appBarBtnSecondary: {
+    color: colors.primary.white,
+  },
+}));
+
+export default function MetricsExplorer() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={4}>
+        <Grid item xs={12}>
+          <CardTitleRow icon={ExploreIcon} label={TITLE} />
+          <ActionTable
+            data={LteMetricsTable}
+            columns={[
+              {title: 'Name', field: 'name'},
+              {title: 'Description', field: 'description'},
+              {title: 'Category', field: 'category'},
+              {title: 'Service', field: 'service'},
+            ]}
+            options={{
+              actionsColumnIndex: -1,
+              pageSizeOptions: [10, 20],
+              pageSize: 20,
+            }}
+            detailPanel={[
+              {
+                icon: ExpandMore,
+                openIcon: ExpandLess,
+                render: rowData => (
+                  <Grafana
+                    grafanaURL={`/grafana?explorer?left=["now-1h","now","Orchestrator Datasource_10", {"expr":"${rowData.promQL}"}]`}
```

Comment:
"Orchestator Datasource_10" being hardcoded is not going to work. This gets into the nitty gritty of how multi-tenant grafana shim works, but I think you'll probably need to do some work in the router to make this possible